context/app/static/js/components/search/SearchViewSwitch.tsx

```diff
@@ -42,6 +42,7 @@ function SearchViewSwitch({ views }: { views: { label: string; icon: typeof SvgI
           id={`${label.toLowerCase()}-view-toggle-button`}
           disabled={view === label.toLowerCase()}
           key={label}
+          aria-label={label.toLowerCase()}
```

Comment:
Same here, was getting `Ensure buttons have discernible text` issue in the dev env, but not prod